Documentation/manual.txt

```diff
@@ -126,6 +126,11 @@ Options:
 	property for each label. The property's name is the label name and the
 	value is the path of the labeled node.
 
+    -L
+	Generates a __local_fixups__ node at the root node. For each property
+	that contains a phandle, the __local_fixups__ node contains a property
+	(at path __local_fixups__/$a if $a is the path of the node). Its value
+	is a list of offsets that are phandle values.
```

Comment:
This doesn't seem quite accurate.  It will generate `__fixups__` as well as `__local_fixups__`, although that's likely not to exist in the cases you use it with.  I think it's worth explicitly pointing out how this interacts with `/plugin/`.